ocs_ci/ocs/ui/views.py

```diff
@@ -1384,6 +1384,33 @@
     ),
     "submariner-custom-source": ("//input[@id='source']", By.XPATH),
     "submariner-custom-channel": ("//input[@id='channel']", By.XPATH),
+    "click-create-policy": ("yaml-create", By.ID),
+    "policy-name": ("#policy-name", By.CSS_SELECTOR),
+    "clear-existing-filter": (
+        "//span[normalize-space()='Clear all filters']",
+        By.XPATH,
+    ),
+    "cluster-selection": (
+        "//tr[td[@data-label='Managed Cluster']/p[text()='{}']]//input[@type='checkbox']",
+        By.XPATH,
+    ),
+    "check-prereq-msg": (
+        "div[class='pf-v5-c-alert pf-m-inline pf-m-success odf-alert mco-create-data-policy__alert'] "
+        "h4[class='pf-v5-c-alert__title']",
```

Comment:
is okay to have patterfly locators?

Reply:
Ideally no, but this was one case where coming up with a stable locator was challenging due to limited properties of the element